atm.js

```diff
@@ -1,21 +1,77 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+//Global Vars
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var currentCheckBalance = 0;
+var currentSaveBalance = 0;
 
-}
+// event listeners
 
-function checkingWithdrawal(amount){
+// checking deposit button
 
-}
+$('#checkDepositButton').click('click', function(){
+    var deposit = $('#checkingInput').val();
+    currentCheckBalance = currentCheckBalance + parseInt(deposit);
+    $('#checkBalanceWindow').text('$' + currentCheckBalance);
 
-function savingsDeposit(amount){
+    // zero balance color change
+    if (currentCheckBalance == 0){
+        $('#checkBalanceWindow').css('background-color', 'red');
```

Comment:
I recommend toggling a class here instead of setting inline styles. If you toggle a class, you can separate your concerns by keeping behavior in JS files and styles in CSS files.